### ISSUE TITLE
[cloud_firestore] Stream connectionState stay waiting until app crashes in iOS Simulator

### ISSUE THREAD

**Issue by rsixel** [FirebaseExtended/flutterfire#4999]
Same on iOS 13.3 iPhone Promax 11 Simulator

flutter doctor:

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel beta, 1.26.0-17.3.pre, on Mac OS X 10.14.6 18G103 darwin-x64, locale pt-BR)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[!] Xcode - develop for iOS and macOS
    ! Xcode 11.3.1 out of date (12.0.1 is recommended).
      Download the latest version or update via the Mac App Store.
[✓] Chrome - develop for the web
[!] Android Studio (not installed)
[✓] VS Code (version 1.53.0)
[✓] Connected device (2 available)
   ....
! Doctor found issues in 2 categories.
```
Snippet:
```
return new StreamBuilder<QuerySnapshot>(
    stream: FirebaseFirestore.instance
        .collection('users/${user.uid}/pedidos')
        .orderBy('entrega')
        .snapshots(),
    builder: (BuildContext context, AsyncSnapshot<QuerySnapshot> snapshot) {
      if (snapshot.hasError) {
        return PlatformText('Something went wrong');
      }

      // Stay entering this if statement several times until crash
      if (snapshot.connectionState == ConnectionState.waiting) {
        return Center(child: PlatformCircularProgressIndicator());
      }


```
pubspec.yml:
```
...
environment:
  sdk: '>=2.12.0-259.8.beta <3.0.0'

dependencies:
  flutter:
    sdk: flutter
  flutter_localizations:
    sdk: flutter

  firebase_core: ^0.8.0-1.0.nullsafety.0
  firebase_core_platform_interface: ^4.0.0-1.0.nullsafety.0
  firebase_auth: ^0.21.0-1.1.nullsafety.1
  cloud_firestore: ^0.17.0-1.0.nullsafety.0

...
```
_Originally posted by @rsixel in https://github.com/FirebaseExtended/flutterfire/issues/4749#issuecomment-774819245_

**Comment by rsixel** [comment]
Checking AsyncSnapshot method   
```
Widget build(BuildContext context, AsyncSnapshot<T> currentSummary) => builder(context, currentSummary);
```
currentSumary.requiredData has the following value:
```
Unhandled exception:
Bad state: Snapshot has neither data nor error
#0      AsyncSnapshot.requireData (package:flutter/src/widgets/async.dart:253:5)
#1      AsyncSnapshot.Eval ()
#2      buildPedidosList.<anonymous closure> (package:gerenciarte/pedido/list.dart:25:16)
#3      StreamBuilder.build (package:flutter/src/widgets/async.dart:545:81)
#4      _StreamBuilderBaseState.build (package:flutter/src/widgets/async.dart:124:48)
#5      StatefulElement.build (package:flutter/src/widgets/framework.dart:4612:27)
#6      ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4495:15)
#7      StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:4667:11)
#8      Element.rebuild (package:flutter/src/widgets/framework.dart:4189:5)
#9      ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4474:5)
#10     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4658:11)
#11     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4469:5)
#12     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3541:14)
#13     Element.updateChild (package:flutter/src/widgets/framework.dart:3306:18)
#14     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5974:14)
#15     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3541:14)
#16     Element.updateChild (package:flutter/src/widgets/framework.dart:3306:18)
#17     SingleChildRenderObjectElement.mount (package:flutter/src/widgets/framework.dart:5974:14)
#18     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3541:14)
#19     Element.updateChild (package:flutter/src/widgets/framework.dart:3306:18)
#20     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4520:16)
#21     Element.rebuild (package:flutter/src/widgets/framework.dart:4189:5)
#22     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4474:5)
#23     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4469:5)
#24     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3541:14)
#25     Element.updateChild (package:flutter/src/widgets/framework.dart:3306:18)
#26     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4520:16)
#27     Element.rebuild (package:flutter/src/widgets/framework.dart:4189:5)
#28     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4474:5)
#29     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4469:5)
#30     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3541:14)
#31     Element.updateChild (package:flutter/src/widgets/framework.dart:3306:18)
#32     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4520:16)
#33     StatefulElement.performRebuild (package:flutter/src/widgets/framework.dart:4667:11)
#34     Element.rebuild (package:flutter/src/widgets/framework.dart:4189:5)
#35     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4474:5)
#36     StatefulElement._firstBuild (package:flutter/src/widgets/framework.dart:4658:11)
#37     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4469:5)
#38     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3541:14)
#39     Element.updateChild (package:flutter/src/widgets/framework.dart:3306:18)
#40     ComponentElement.performRebuild (package:flutter/src/widgets/framework.dart:4520:16)
#41     Element.rebuild (package:flutter/src/widgets/framework.dart:4189:5)
#42     ComponentElement._firstBuild (package:flutter/src/widgets/framework.dart:4474:5)
#43     ComponentElement.mount (package:flutter/src/widgets/framework.dart:4469:5)
#44     Element.inflateWidget (package:flutter/src/widgets/framework.dart:3541:14)
#45     Element.updateChild (package:flutter/src/widgets/framework.dart:3306:18)
#46     SliverMultiBoxAdaptorElement.updateChild (package:flutter/src/widgets/sliver.dart:1229:37)
#47     SliverMultiBoxAdaptorElement.createChild.<anonymous closure> (package:flutter/src/widgets/sliver.dart:1214:20)
#48     BuildOwner.buildScope (package:flutter/src/widgets/framework.dart:2647:19)
#49     SliverMultiBoxAdaptorElement.createChild (package:flutter/src/widgets/sliver.dart:1207:12)
#50     RenderSliverMultiBoxAdaptor._createOrObtainChild.<anonymous closure> (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:349:23)
#51     RenderObject.invokeLayoutCallback.<anonymous closure> (package:flutter/src/rendering/object.dart:1894:59)
#52     PipelineOwner._enableMutationsToDirtySubtrees (package:flutter/src/rendering/object.dart:915:15)
#53     RenderObject.invokeLayoutCallback (package:flutter/src/rendering/object.dart:1894:14)
#54     RenderSliverMultiBoxAdaptor._createOrObtainChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:338:5)
#55     RenderSliverMultiBoxAdaptor.addInitialChild (package:flutter/src/rendering/sliver_multi_box_adaptor.dart:422:5)
#56     RenderSliverFixedExtentBoxAdaptor.performLayout (package:flutter/src/rendering/sliver_fixed_extent_list.dart:205:12)
#57     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#58     RenderSliverEdgeInsetsPadding.performLayout (package:flutter/src/rendering/sliver_padding.dart:137:12)
#59     _RenderSliverFractionalPadding.performLayout (package:flutter/src/widgets/sliver_fill.dart:167:11)
#60     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#61     RenderViewportBase.layoutChildSequence (package:flutter/src/rendering/viewport.dart:512:13)
#62     RenderViewport._attemptLayout (package:flutter/src/rendering/viewport.dart:1570:12)
#63     RenderViewport.performLayout (package:flutter/src/rendering/viewport.dart:1479:20)
#64     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#65     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#66     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#67     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#68     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#69     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#70     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#71     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#72     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#73     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#74     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#75     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#76     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#77     RenderPadding.performLayout (package:flutter/src/rendering/shifted_box.dart:226:12)
#78     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#79     ChildLayoutHelper.layoutChild (package:flutter/src/rendering/layout_helper.dart:54:11)
#80     RenderStack._computeSize (package:flutter/src/rendering/stack.dart:570:43)
#81     RenderStack.performLayout (package:flutter/src/rendering/stack.dart:597:12)
#82     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#83     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#84     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#85     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#86     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#87     RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#88     RenderOffstage.performLayout (package:flutter/src/rendering/proxy_box.dart:3362:13)
#89     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#90     ChildLayoutHelper.layoutChild (package:flutter/src/rendering/layout_helper.dart:54:11)
#91     RenderStack._computeSize (package:flutter/src/rendering/stack.dart:570:43)
#92     RenderStack.performLayout (package:flutter/src/rendering/stack.dart:597:12)
#93     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#94     RenderPadding.performLayout (package:flutter/src/rendering/shifted_box.dart:226:12)
#95     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#96     ChildLayoutHelper.layoutChild (package:flutter/src/rendering/layout_helper.dart:54:11)
#97     RenderStack._computeSize (package:flutter/src/rendering/stack.dart:570:43)
#98     RenderStack.performLayout (package:flutter/src/rendering/stack.dart:597:12)
#99     RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#100    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#101    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#102    ChildLayoutHelper.layoutChild (package:flutter/src/rendering/layout_helper.dart:54:11)
#103    RenderStack._computeSize (package:flutter/src/rendering/stack.dart:570:43)
#104    RenderStack.performLayout (package:flutter/src/rendering/stack.dart:597:12)
#105    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#106    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#107    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#108    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#109    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#110    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#111    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#112    ChildLayoutHelper.layoutChild (package:flutter/src/rendering/layout_helper.dart:54:11)
#113    RenderStack._computeSize (package:flutter/src/rendering/stack.dart:570:43)
#114    RenderStack.performLayout (package:flutter/src/rendering/stack.dart:597:12)
#115    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#116    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#117    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#118    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#119    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#120    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#121    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#122    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#123    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#124    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#125    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#126    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#127    RenderOffstage.performLayout (package:flutter/src/rendering/proxy_box.dart:3362:13)
#128    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#129    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#130    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#131    _RenderTheatre.performLayout (package:flutter/src/widgets/overlay.dart:743:15)
#132    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#133    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#134    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#135    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#136    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#137    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#138    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#139    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#140    RenderCustomPaint.performLayout (package:flutter/src/rendering/custom_paint.dart:546:11)
#141    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#142    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#143    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#144    RenderProxyBoxMixin.performLayout (package:flutter/src/rendering/proxy_box.dart:118:14)
#145    RenderObject.layout (package:flutter/src/rendering/object.dart:1784:7)
#146    RenderView.performLayout (package:flutter/src/rendering/view.dart:153:14)
#147    RenderObject._layoutWithoutResize (package:flutter/src/rendering/object.dart:1641:7)
#148    PipelineOwner.flushLayout (package:flutter/src/rendering/object.dart:884:18)
#149    RendererBinding.drawFrame (package:flutter/src/rendering/binding.dart:453:19)
#150    WidgetsBinding.drawFrame (package:flutter/src/widgets/binding.dart:874:13)
#151    RendererBinding._handlePersistentFrameCallback (package:flutter/src/rendering/binding.dart:319:5)
#152    SchedulerBinding._invokeFrameCallback (package:flutter/src/scheduler/binding.dart:1144:15)
#153    SchedulerBinding.handleDrawFrame (package:flutter/src/scheduler/binding.dart:1082:9)
#154    SchedulerBinding.scheduleWarmUpFrame.<anonymous closure> (package:flutter/src/scheduler/binding.dart:865:7)
#155    _rootRun (dart:async/zone.dart:1346:47)
#156    _CustomZone.run (dart:async/zone.dart:1258:19)
#157    _CustomZone.runGuarded (dart:async/zone.dart:1162:7)
#158    _CustomZone.bindCallbackGuarded.<anonymous closure> (dart:async/zone.dart:1202:23)
#159    _rootRun (dart:async/zone.dart:1354:13)
#160    _CustomZone.run (dart:async/zone.dart:1258:19)
#161    _CustomZone.bindCallback.<anonymous closure> (dart:async/zone.dart:1186:23)
#162    Timer._createTimer.<anonymous closure> (dart:async-patch/timer_patch.dart:18:15)
#163    _Timer._runTimers (dart:isolate-patch/timer_impl.dart:395:19)
#164    _Timer._handleMessage (dart:isolate-patch/timer_impl.dart:426:5)
#165    _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:184:12)
```

**Comment by markusaksli-nc** [comment]
Hi @rsixel 
Could you please update your outdated Xcode version to the latest `12.4`?
Could you please also provide your `flutter doctor -v` and `flutter pub deps -- --style=compact`?
Thank you

**Comment by rsixel** [comment]
About XCode, my last available upgrade is macOs is Mojave 10.14.6 due to my old macbook pro. It forces me to stick to 11.3.1.
BTW , stable channel using the same iOS SDK version (AKA 7.3.0) is doing fine.

So follow the information you need:

flutter doctor -v
```
Framework • revision 1fe38dcb5f (2 days ago) • 2021-02-10 16:25:47 -0800
Engine • revision d4453f6018
Tools • Dart 2.12.0 (build 2.12.0-259.9.beta)

Running flutter doctor...
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel beta, 1.26.0-17.5.pre, on Mac OS X 10.14.6 18G103 darwin-x64, locale pt-BR)
[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
[!] Xcode - develop for iOS and macOS
    ! Xcode 11.3.1 out of date (12.0.1 is recommended).
      Download the latest version or update via the Mac App Store.
[✓] Chrome - develop for the web
[!] Android Studio (not installed)
[✓] VS Code (version 1.53.2)
[✓] Connected device (2 available)
    ! Error: iPad de Ricardo is not connected. Xcode will continue when iPad de Ricardo is connected. (code -13)
    ! Error: iPhone de Ricardo is not connected. Xcode will continue when iPhone de Ricardo is connected. (code -13)

! Doctor found issues in 2 categories.
MacBook-Pro-de-Ricardo:gerenciarte_app ricardo$ flutter doctor -v
[✓] Flutter (Channel beta, 1.26.0-17.5.pre, on Mac OS X 10.14.6 18G103 darwin-x64, locale pt-BR)
    • Flutter version 1.26.0-17.5.pre at /Volumes/HDD/dev/flutter
    • Framework revision 1fe38dcb5f (2 days ago), 2021-02-10 16:25:47 -0800
    • Engine revision d4453f6018
    • Dart version 2.12.0 (build 2.12.0-259.9.beta)

[✓] Android toolchain - develop for Android devices (Android SDK version 29.0.2)
    • Android SDK at /Volumes/HDD/dev/android/sdk
    • Platform android-30, build-tools 29.0.2
    • ANDROID_HOME = /Volumes/HDD/dev/android/sdk
    • ANDROID_SDK_ROOT = /Volumes/HDD/dev/android/sdk
    • Java binary at: /Library/Java/JavaVirtualMachines/jdk1.8.0_192.jdk/Contents/Home/bin/java
    • Java version Java(TM) SE Runtime Environment (build 1.8.0_192-b12)
    • All Android licenses accepted.

[!] Xcode - develop for iOS and macOS
    • Xcode at /Volumes/HDD/Xcode.app/Contents/Developer
    • Xcode 11.3.1, Build version 11C505
    ! Xcode 11.3.1 out of date (12.0.1 is recommended).
      Download the latest version or update via the Mac App Store.
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).

[✓] VS Code (version 1.53.2)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.19.0

[✓] Connected device (2 available)
    • AOSP on IA Emulator (mobile) • emulator-5554 • android-x86    • Android 9 (API 28) (emulator)
    • Chrome (web)                 • chrome        • web-javascript • Google Chrome 88.0.4324.150
    • iPhone 11 Pro Max (mobile) • 2655A99B-F289-434F-B3D5-B9FFF032F0E4 • ios            • com.apple.CoreSimulator.SimRuntime.iOS-13-3
      (simulator)
  
! Doctor found issues in 2 categories.
```

 flutter pub deps -- --style=compact
```
Dart SDK 2.12.0-259.9.beta
Flutter SDK 1.26.0-17.5.pre
(...)
dependencies:
- cloud_firestore 0.17.0-1.0.nullsafety.0 [cloud_firestore_platform_interface cloud_firestore_web firebase_core firebase_core_platform_interface flutter meta]
- cupertino_list_tile 0.2.0-nullsafety.0 [cupertino_icons flutter]
- cupertino_radio_choice 1.0.1+1 [flutter]
- enum_to_string 1.0.14
- firebase_auth 0.21.0-1.1.nullsafety.1 [firebase_auth_platform_interface firebase_auth_web firebase_core firebase_core_platform_interface flutter meta]
- firebase_core 0.8.0-1.0.nullsafety.1 [firebase_core_platform_interface firebase_core_web flutter meta]
- firebase_ui 1.0.8 [flutter flutter_twitter_login firebase_auth google_sign_in flutter_facebook_login apple_sign_in]
- flutter 0.0.0 [characters collection meta typed_data vector_math sky_engine]
- flutter_localizations 0.0.0 [flutter intl characters clock collection meta path typed_data vector_math]
- flutter_masked_text 0.8.1 [flutter]
- flutter_platform_widgets 1.0.0-nullsafety.0 [flutter]
- intl 0.17.0 [clock path]
- material_search 0.2.8 [mockito meta flutter]
- open_file 4.0.0-nullsafety.0 [flutter ffi]
- path_provider 2.0.0-nullsafety [flutter path_provider_platform_interface path_provider_macos path_provider_linux path_provider_windows]
- pdf 3.0.0-nullsafety.1 [archive barcode crypto image meta path_parsing vector_math xml]
- printing 5.0.0-nullsafety.1 [flutter flutter_web_plugins image js meta pdf plugin_platform_interface]

dev dependencies:
- flutter_test 0.0.0 [flutter test_api path fake_async clock stack_trace vector_math async boolean_selector characters charcode collection matcher meta source_span stream_channel string_scanner term_glyph typed_data]

transitive dependencies:
- _fe_analyzer_shared 14.0.0 [meta]
- analyzer 0.41.2 [_fe_analyzer_shared args cli_util collection convert crypto glob meta package_config path pub_semver source_span watcher yaml]
- apple_sign_in 0.1.0 [flutter]
- archive 3.0.0-nullsafety.0 [crypto path]
- args 2.0.0
- async 2.5.0 [collection]
- barcode 2.0.0-nullsafety [meta qr]
- boolean_selector 2.1.0 [source_span string_scanner]
- characters 1.1.0
- charcode 1.2.0
- cli_util 0.2.0 [path]
- clock 1.1.0
- cloud_firestore_platform_interface 4.0.0-1.0.nullsafety.0 [collection firebase_core flutter meta plugin_platform_interface]
- cloud_firestore_web 0.4.0-1.0.nullsafety.0 [cloud_firestore_platform_interface firebase_core firebase_core_web flutter flutter_web_plugins js]
- collection 1.15.0
- convert 2.1.1 [charcode typed_data]
- coverage 0.15.2 [args logging package_config path source_maps stack_trace vm_service]
- crypto 3.0.0-nullsafety.0 [collection typed_data]
- cupertino_icons 1.0.2
- fake_async 1.2.0 [clock collection]
- ffi 0.3.0-nullsafety.1
- file 6.0.0-nullsafety.4 [meta path]
- firebase_auth_platform_interface 4.0.0-1.1.nullsafety.1 [firebase_core flutter meta plugin_platform_interface]
- firebase_auth_web 0.4.0-1.1.nullsafety.1 [firebase_auth_platform_interface firebase_core firebase_core_web flutter flutter_web_plugins http_parser intl js meta]
- firebase_core_platform_interface 4.0.0-1.0.nullsafety.1 [flutter meta plugin_platform_interface]
- firebase_core_web 0.3.0-1.0.nullsafety.1 [firebase_core_platform_interface flutter flutter_web_plugins js meta]
- flutter_facebook_login 3.0.0 [flutter collection]
- flutter_twitter_login 1.1.0 [flutter]
- flutter_web_plugins 0.0.0 [flutter js characters collection meta typed_data vector_math]
- glob 1.2.0 [async collection node_io path pedantic string_scanner]
- google_sign_in 5.0.0-nullsafety [google_sign_in_platform_interface flutter meta]
- google_sign_in_platform_interface 2.0.0-nullsafety [flutter meta quiver]
- http_multi_server 2.2.0 [async]
- http_parser 4.0.0-nullsafety [charcode collection source_span string_scanner typed_data]
- image 3.0.0-nullsafety.0 [archive xml meta]
- io 0.3.4 [charcode meta path string_scanner]
- js 0.6.3
- logging 1.0.0
- matcher 0.12.10 [stack_trace]
- meta 1.3.0
- mime 1.0.0
- mockito 3.0.2 [collection matcher meta test]
- node_interop 1.2.1 [js]
- node_io 1.1.1 [node_interop path]
- node_preamble 1.4.13
- package_config 1.9.3 [path charcode]
- path 1.8.0
- path_parsing 0.2.0-nullsafety.0 [vector_math meta]
- path_provider_linux 0.2.0-nullsafety [path xdg_directories path_provider_platform_interface flutter]
- path_provider_macos 0.0.5-nullsafety [flutter]
- path_provider_platform_interface 2.0.0-nullsafety [flutter meta platform plugin_platform_interface]
- path_provider_windows 0.1.0-nullsafety.2 [path_provider_platform_interface meta path flutter ffi win32]
- pedantic 1.10.0
- petitparser 4.0.0-nullsafety.1 [meta]
- platform 3.0.0
- plugin_platform_interface 1.1.0-nullsafety.2 [meta]
- pool 1.5.0 [async stack_trace]
- process 4.0.0-nullsafety.4 [file path platform]
- pub_semver 1.4.4 [collection]
- qr 2.0.0-nullsafety.1 [meta]
- quiver 3.0.0-nullsafety.3 [matcher]
- shelf 1.0.0-nullsafety.0 [async collection http_parser path stack_trace stream_channel]
- shelf_packages_handler 2.0.1 [path shelf shelf_static]
- shelf_static 0.2.9+2 [convert http_parser mime path shelf]
- shelf_web_socket 0.2.4 [shelf stream_channel web_socket_channel]
- sky_engine 0.0.99
- source_map_stack_trace 2.1.0 [path stack_trace source_maps]
- source_maps 0.10.10 [source_span]
- source_span 1.8.0 [charcode collection path term_glyph]
- stack_trace 1.10.0 [path]
- stream_channel 2.1.0 [async]
- string_scanner 1.1.0 [charcode source_span]
- term_glyph 1.2.0
- test 1.16.0 [analyzer async boolean_selector coverage http_multi_server io js node_preamble package_config path pedantic pool shelf shelf_packages_handler shelf_static shelf_web_socket source_span stack_trace stream_channel typed_data web_socket_channel webkit_inspection_protocol yaml test_api test_core]
- test_api 0.2.19 [async boolean_selector collection meta path source_span stack_trace stream_channel string_scanner term_glyph matcher]
- test_core 0.3.12 [analyzer async args boolean_selector collection coverage glob io meta package_config path pedantic pool source_map_stack_trace source_maps source_span stack_trace stream_channel vm_service yaml matcher test_api]
- typed_data 1.3.0 [collection]
- vector_math 2.1.0
- vm_service 6.0.1
- watcher 0.9.7+15 [async path pedantic]
- web_socket_channel 1.2.0 [async crypto stream_channel]
- webkit_inspection_protocol 0.7.5 [logging]
- win32 2.0.0-nullsafety.10 [ffi]
- xdg_directories 0.2.0-nullsafety.1 [meta path process]
- xml 5.0.0-nullsafety.1 [collection meta petitparser]
- yaml 2.2.1 [charcode collection string_scanner source_span]
```

I run the project using flutter run --no-sound-null-safety -d "iPhone 11 Pro Max"

Regards!

**Comment by markusaksli-nc** [comment]
Oh since the stable version works this can just be included in https://github.com/FirebaseExtended/flutterfire/issues/4948

The old Xcode and iOS versions shouldn't be that important if they worked before.

Closing this as a duplicate of https://github.com/FirebaseExtended/flutterfire/issues/4948 so it would be easier to track. Could you please comment the crash info there as well?
